<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IQVisualizer</class>
 <widget class="QWidget" name="IQVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="channalAnalyzerWidget_groupBox">
     <property name="title">
      <string>Название канала</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Параметры сигнала</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>F (Hz)</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Amax</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Amid</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="signalParametrs_maxAmplitude_label">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="signalParametrs_frequency_label">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="signalParametrs_middleAmplitude_label">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Параметры отображения</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Частота обновления (Hz)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="visibleParametrs_refreshFrequency_comboBox">
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>5</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>20</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>50</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Размер точек</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="visibleParametrs_pointsSize_horizontalSlider">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="visibleParametrs_line_checkBox">
                 <property name="text">
                  <string>Линии</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCustomPlot" name="plot_widget" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0,0,255,100);
</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="IQWaterfall" name="waterfall_widget" native="true"/>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="mode_amplitude_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/A.svg</normaloff>:/Icons/A.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mode_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mode_frequency_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/F.svg</normaloff>:/Icons/F.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mode_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mode_spectrum_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/Spectr.svg</normaloff>:/Icons/Spectr.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mode_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mode_constellation_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/Star.svg</normaloff>:/Icons/Star.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mode_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="nvl_label">
              <property name="text">
               <string>NVL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="nvl_comboBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>128</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>256</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>512</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1024</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2048</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4096</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8192</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="play_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/play.svg</normaloff>:/Icons/play.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">channalState_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/stop.svg</normaloff>:/Icons/stop.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">channalState_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="autoRescale_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/Rescale.svg</normaloff>:/Icons/Rescale.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="waterfall_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/Waterfall.svg</normaloff>:/Icons/Waterfall.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="settings_pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourses.qrc">
                <normaloff>:/Icons/Settings.svg</normaloff>:/Icons/Settings.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Настройки канала</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>UDP порт</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QSpinBox" name="srcUdpPort_spinBox">
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>38100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="iqType_uint16_pushButton">
              <property name="text">
               <string>UINT16</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">iqType_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Тип IQ</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="iqType_uint8_pushButton">
              <property name="text">
               <string>UINT8</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">iqType_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="iqType_pcm24_pushButton">
              <property name="text">
               <string>PCM24</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">iqType_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="1" colspan="3">
             <widget class="QSpinBox" name="iqStartNumberByte_spinBox">
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>36</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Номер байта начала отсчётов</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>206</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="apply_pushButton">
              <property name="text">
               <string>Применить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancel_pushButton">
              <property name="text">
               <string>Отмена</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">IQVisualizer/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IQWaterfall</class>
   <extends>QWidget</extends>
   <header>IQVisualizer/iqwaterfall.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="iqType_buttonGroup"/>
  <buttongroup name="mode_buttonGroup"/>
  <buttongroup name="channalState_buttonGroup"/>
 </buttongroups>
</ui>
